sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -163,3 +163,21 @@ export class ApplicationSummaryAPIOutDTO {
   submitted?: Date;
   status: ApplicationStatus;
 }
+
+/**
+ * History of SIN validations associated with a user.
+ */
+export class SINValidationsAPIOutDTO {
+  id: number;
+  sin: string;
+  createdAt: Date;
+  isValidSIN?: boolean;
+  sinStatus?: string;
+  validSINCheck?: string;
+  validBirthdateCheck?: string;
+  validFirstNameCheck?: string;
+  validLastNameCheck?: string;
+  validGenderCheck?: string;
+  temporarySIN: boolean;
```

Comment:
Not major, but I think the name here should be like isSINTemporary. 
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig